### DISCUSSION TITLE
API for ECSQL

### DISCUSSION THREAD

**Discussion by madibai** [iTwin/itwinjs-core#3758] [General]
Hi
How can i connect to ecsql via api and run sql some query ? 
Is any sample or tutorial ?
thx

**Comment by aruniverse** [comment]
Are you trying to query ecsql in an app that uses an iTwin.js Viewer or? Are you hoping to query in a frontend or backend app?

**Reply by ayushibasaiwal** [reply]
@madibai Can you please elaborate and answer the above questions?
